### PR TITLE
MISUV-7149 KEL756 - Balancing Payment Label Missing

### PR DESCRIPTION
**Pull request by tharunprasath**

### CODE REVIEW

File: app/views/TaxYearSummary.scala.html [hunk 1]
```diff
@@ -209,8 +209,12 @@ <h2 class="govuk-heading-m">@messages(header)</h2>
                             (documentDetailWithDueDate.isLatePaymentInterest && !documentDetailWithDueDate.documentDetail.interestIsPaid))) {
                             <div class="govuk-tag govuk-tag--red">@messages("tax-year-summary.payments.overdue")</div>
                         }
-                        <div>
-                            <a class="govuk-link"  id="paymentTypeLink-@{index}" href="@getChargeSummaryHref(documentDetailWithDueDate)">@getChargeSummaryMessage(documentDetailWithDueDate)</a>
+                        <div id="paymentTypeText-@{index}">
+                            @if(documentDetailWithDueDate.documentDetail.originalAmount.getOrElse(throw MissingFieldException("Original Amount")) == 0) {
```
Comment (avelx) [inline app/views/TaxYearSummary.scala.html:213]: I would suspect that we already have a story to refactor / remove code where we throwing exceptions in the view ( https://jira.tools.tax.service.gov.uk/browse/MISUV-6912 ), if not we need to create a story for this

File: test/views/TaxYearSummaryViewSpec.scala [hunk 2]
```diff
@@ -751,14 +751,13 @@ class TaxYearSummaryViewSpec extends ViewSpec with FeatureSwitching {
...
         val paymentTypeLink: Element = layoutContent.getElementById("paymentTypeLink-0")
         val paymentTabRow: Element = layoutContent.getElementById("payments-table").getElementsByClass("govuk-table__row").get(1)
         paymentTabRow.getElementsByClass("govuk-table__cell").first().text() shouldBe "N/A"
         paymentTabRow.getElementsByClass("govuk-table__cell").get(1).text() shouldBe BigDecimal(0).toCurrencyString
-        paymentTypeLink.text shouldBe remainingBalance
-        paymentTypeLink.attr("href") shouldBe controllers.routes.ChargeSummaryController.show(
-          testYear, fullDocumentDetailModel.transactionId).url
-
+        paymentTypeText.text shouldBe remainingBalance
+        paymentTypeLink shouldBe null
```
Comment (avelx) [inline test/views/TaxYearSummaryViewSpec.scala:760]: please do not use "null" in our code base

**Review by avelx — CHANGES_REQUESTED** [review]
Remove null usage

File: app/views/TaxYearSummary.scala.html (hunk 1 above)
Comment (tharunprasath) [reply app/views/TaxYearSummary.scala.html:213]: Created one. https://jira.tools.tax.service.gov.uk/browse/MISUV-7154

**Pull request merged**